performance_manager/pyproject.toml

```diff
@@ -53,4 +55,7 @@ disallow_untyped_defs = true
 warn_unused_ignores = true
 warn_unreachable = true
 pretty = true
-plugins = "sqlmypy"
+plugins = [
+  "sqlalchemy.ext.mypy.plugin"
+]
```

Comment:
i changed which sqalchemy mypy plugin we use from dropbox's custom one to sqlalchemy's shipped one. it solves some problems but creates other. it feels like it will make the 2.0 switch easier whenever that happens.